packages/web/src/fixtures/dev-kit/client.ts

```diff
@@ -1,17 +1,46 @@
 import { contractFactory } from '@devprtcl/dev-kit-js'
+import { addresses } from '@devprtcl/dev-kit-js'
+import { getAccountAddress } from 'src/fixtures/wallet/utility'
 
-export const newClient = () => {
+const newClient = () => {
   const { ethereum } = window
   if (ethereum) {
     return contractFactory(ethereum)
   }
   return undefined
 }
 
-export const getAccountAddress = () => {
-  const { ethereum } = window
-  if (ethereum) {
-    return ethereum.selectedAddress
+export const getRewardsAmount = async (propertyAddress: string) => {
+  const client = newClient()
+  if (client) {
```

Comment:
If you want to return undefined, you can write it more concisely with `Optional Chaining`.  

However, I don't think you have to force yourself to fix it.  I think the readability is high because the description is aligned in the whole of this file.

Up to you.